boards/arm/mec172xevb_assy6906/doc/index.rst

```diff
@@ -0,0 +1,392 @@
++----------+----------+--------+-----------+----------+---------+
+|  10-11   |  10-11   |        | 11-12     | 11-12    |         |
++----------+----------+--------+-----------+----------+---------+
+|          |          |        | 14-15     | 14-15    |         |
++----------+----------+--------+-----------+----------+---------+
+|          |          |        | 17-18     | 20-21    |         |
++----------+----------+--------+-----------+----------+---------+
+
+.. note:: For UART1 make sure JP39 have jumpers connected 1-2, 3-4.
+
+To receive UART1 serial output, please refer to the picture below
+to make sure that JP9 configured for UART1 output.
+
+Jumper settings for MEC172x 176WFBGA Socket DC Assy 6915 Rev B1p0
+=================================================================
+
+The jumper configuration explained above covers the base board. The ASSY
+6915 MEC1723 CPU board provides capability for an optional, external 32KHz
+clock source. The card includes a 32KHz crystal oscillator. The card can
+also be configured to use an external 50% duty cycle 32KHz source on the
+XTAL2/32KHZ_IN pin. Note, firmware must set the MEC172x clock enable
+register to select the external source matching the jumper settings. If
+using the MEC15xx internal silicon oscillator then the 32K jumper settings
+are don't cares. ``JP1`` is for scoping test clock outputs. Please refer to
+the schematic in reference section below.
+
+Parallel 32KHz crystal configuration
+------------------------------------
++-------+-------+
+| JP2   | JP3   |
++=======+=======+
+| 1-2   | 2-3   |
++-------+-------+
+
+External 32KHz 50% duty cycle configuration
+-------------------------------------------
++-------+-------+
+| JP2   | JP3   |
++=======+=======+
+| NC    | 1-2   |
++-------+-------+
+
+
+Jumper settings for MEC1723 176WFBGA Socket DC Assy 6915 Rev B1p0
+=================================================================
+
+The MEC1723 ASSY 6915 CPU card does not include an onboard external
+32K crystal or oscillator. The one jumper block ``JP1`` is for scoping
+test clock outputs not for configuration. Please refer to schematic
+in reference section below.
+
+Programming and Debugging
+*************************
+
+Setup
+=====
+#. If you use Dediprog SF100 programmer, then setup it.
+
+   Windows version can be found at the `SF100 Product page`_.
+
+   Linux version source code can be found at `SF100 Linux GitHub`_.
+   Follow the `SF100 Linux manual`_ to complete setup of the SF100 programmer.
+   For Linux please make sure that you copied ``60-dediprog.rules``
+   from the ``SF100Linux`` folder to the :code:`/etc/udev/rules.s` (or rules.d)
+   then restart service using:
+
+   .. code-block:: console
+
+      $ udevadm control --reload
+
+   Add directory with program ``dpcmd`` (on Linux)
+   or ``dpcmd.exe`` (on Windows) to your ``PATH``.
+
+#. Clone the `MEC172x SPI Image Gen`_ repository or download the files within
```

Comment:
can it be turned into a zephyr module? that way build system can just do this transparently?

Reply:
This is a good idea, perhaps as enhancement https://github.com/zephyrproject-rtos/zephyr/issues/37053